os/adding-swap.md

```diff
@@ -0,0 +1,74 @@
+# Adding swap in CoreOS
+
+Swap is the process of moving pages of memory to a designated part of the hard disk, freeing up space when needed. Swap can be used to alleviate problems with low-memory environments.
+
+By default CoreOS does not include a partition for swap, however one can configure their system to have swap.
+
+## Manually adding a swapfile on ext4
+
+Run these commands **as root** to create a 512 megabyte swapfile.
+
+```
+$ fallocate -l 512m /media/state/512MiB.swap
```

Comment:
`/media/state` is an extremely old construct of CoreOS. (where did you find this?) Just use /var instead.